ext_localconf.php

```diff
@@ -9,3 +9,31 @@
 
 $GLOBALS['TYPO3_CONF_VARS']['SC_OPTIONS']['extbase']['commandControllers'][] =
     \NamelessCoder\AsyncReferenceIndexing\Command\ReferenceIndexCommandController::class;
+
+// Overwrite lowlevel_refindex since we want to perform the indexing and not spool into our table
```

Comment:
OK :-) You will be allowed to consider it further - on the other hand we should then a least write a note on it if we decide to keep it the way it is :-)